tests/baselines/reference/tscWatch/programUpdates/updates-errors-when-forceConsistentCasingInFileNames-changes.js

```diff
@@ -27,7 +27,21 @@ Output::
 >> Screen clear
 [[90m12:00:15 AM[0m] Starting compilation in watch mode...
 
-[[90m12:00:20 AM[0m] Found 0 errors. Watching for file changes.
+[96mb.ts[0m:[93m1[0m:[93m43[0m - [91merror[0m[90m TS1149: [0mFile name '/A.ts' differs from already included file name '/a.ts' only in casing.
```

Comment:
whats this diff?

Reply:
Oh because default changed. can you change the test to set forceConsistenCasing to true and false instead of using default so it is actually checking what the test is suppose to check.